python/chronos/src/bigdl/chronos/forecaster/tcn_forecaster.py

```diff
@@ -69,6 +70,9 @@ def __init__(self,
         :param future_seq_len: Specify the output time steps (i.e. horizon).
         :param input_feature_num: Specify the feature dimension.
         :param output_feature_num: Specify the output dimension.
+        :param dummy_encoder: bool, no encoder is applied if True, which will
```

Comment:
Maybe we need to add some assert code, to make sure the input_feature_num equals to output_feature_num when dummy_encoder is True.